Solutions/Corvus.Extensions.Newtonsoft.Json.Specs/JsonExtensionsSpecsSteps.cs

```diff
@@ -37,12 +37,14 @@ public JsonExtensionsSpecsSteps(FeatureContext featureContext, ScenarioContext s
         {
             this.featureContext = featureContext;
             this.scenarioContext = scenarioContext;
-            this.propertyBagFactory = ContainerBindings.GetServiceProvider(featureContext).GetService<IPropertyBagFactory>();
-            this.jnetPropertyBagFactory = ContainerBindings.GetServiceProvider(featureContext).GetService<IJsonNetPropertyBagFactory>();
+            this.propertyBagFactory = ContainerBindings.GetServiceProvider(featureContext).GetService<IPropertyBagFactory>()!;
```

Comment:
Better to use `GetRequiredService` here instead of ignoring the possibility of a null result.

Reply:
Done